openslides_backend/action/actions/committee/update.py

```diff
@@ -36,49 +37,33 @@ class CommitteeUpdateAction(UpdateAction):
     )
 
     def check_permissions(self, instance: Dict[str, Any]) -> None:
-        is_manager = has_committee_management_level(
-            self.datastore,
-            self.user_id,
-            CommitteeManagementLevel.CAN_MANAGE,
-            instance["id"],
-        )
-        can_manage_organization = has_organization_management_level(
+        if has_organization_management_level(
             self.datastore,
             self.user_id,
             OrganizationManagementLevel.CAN_MANAGE_ORGANIZATION,
-        )
-        if (
-            any(
-                [
-                    field in instance
-                    for field in [
-                        "name",
-                        "description",
-                        "template_meeting_id",
-                        "default_meeting_id",
-                    ]
-                ]
-            )
-            and not is_manager
         ):
-            raise MissingPermission(CommitteeManagementLevel.CAN_MANAGE)
-        if (
-            any(
-                [
-                    field in instance
-                    for field in [
-                        "user_ids",
-                        "forward_to_committee_ids",
-                        "receive_forwardings_from_committee_ids",
-                    ]
+            return
+
+        if any(
+            [
+                field in instance
+                for field in [
+                    "user_ids",
+                    "forward_to_committee_ids",
+                    "receive_forwardings_from_committee_ids",
                 ]
-            )
-            and not can_manage_organization
+            ]
         ):
             raise MissingPermission(OrganizationManagementLevel.CAN_MANAGE_ORGANIZATION)
-        if (
-            "organization_tag_ids" in instance
-            and not is_manager
-            and not can_manage_organization
+
+        if has_committee_management_level(
+            self.datastore,
+            self.user_id,
+            CommitteeManagementLevel.CAN_MANAGE,
+            instance["id"],
         ):
-            raise PermissionDenied("Missing can_manage_organization and not manager.")
+            return
+
+        raise PermissionDenied(
+            "Missing can_manage_organization or can_manage_committee."
```

Comment:
You can throw a `MissingPermission` here and just provide multiple missing permissions in a map.